### PR TITLE
Pre-training script

### PR DESCRIPTION
**Pull request by mayurbhangale**
script to download and prepare dataset for traininig

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # bert-cosine-sim
 Fine-tune BERT to generate sentence embedding for cosine similarity. Most of the code is copied from [huggingface's bert project](https://github.com/huggingface/pytorch-pretrained-BERT).
 
+# Preparing data for training
+
+`python prerun.py` downloads, extracts and saves model and training data (STS-B) in relevant folder, after which you can simplu modify hyperparameters in `run.sh`
```
Comment (beekbin) [inline README.md]: typo: `simplu--> simply`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # bert-cosine-sim
 Fine-tune BERT to generate sentence embedding for cosine similarity. Most of the code is copied from [huggingface's bert project](https://github.com/huggingface/pytorch-pretrained-BERT).
 
+# Preparing data for training
```
Comment (beekbin) [inline README.md]: `Preparing data for training`  --> `Download data and pre-trained model for fine-tuning`

File: prerun.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import urllib.request
+import os
+import zipfile
+import pandas as pd
+import numpy as np
+import pickle
+
+data_dir = 'data/'
```
Comment (beekbin) [inline prerun.py:8]: It would be nicer to create two directories: one for data, and the other for the pre-trained model.

File: prerun.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    urllib.request.urlretrieve(bert_vocab, data_dir+'vocab.txt')
+    print("Saved vocab")
+
+    print("Downloading and extracting STS-B")
+    stsb_zip = data_dir+'sts_b.zip'
+    urllib.request.urlretrieve(stsb_dataset, stsb_zip)
+    with zipfile.ZipFile(stsb_zip) as zip_ref:
+        zip_ref.extractall(data_dir)
+    os.remove(stsb_zip)
+    print("\tCompleted!")
```
Comment (beekbin) [inline prerun.py:41]: Add one line to execute the function:
 ```downloade_models()```

**Review by beekbin — CHANGES_REQUESTED** [review]
Hi Mayur, thanks for this PR, it will make this repo more useful.

**Comment by mayurbhangale** [comment]
Updated, Please review

**Review by beekbin — APPROVED** [review]
Thanks a lot for your contributions!

**Comment by mayurbhangale** [comment]
Please merge this to master

**Pull request merged**